src/openforms/registrations/contrib/objects_api/templatetags/registrations/contrib/objects_api/json_tags.py

```diff
@@ -32,3 +31,17 @@ def json_summary(context):
         json_data = html_escape_json(json_data)
 
     return SafeString(json.dumps(json_data))
+
+
+@register.simple_tag
+def as_geo_json(value: list[float] | str) -> SafeString | dict:
+    """Output the ``value`` as a safe GeoJSON dumped string.
+
+    As of today, this only supports coordinates. This essentially does the same thing as
+    :func:`_transform_coordinates`, but for any map component.
+    """
+    if not value:
+        return {}
```

Comment:
shouldn't this be `SafeString(json.dumps({}))`?

Now the string representation of an empty dict "happens" to be the same as an empty object in JSON, but that's a coincidence, and that would make the tag return a consistent type instead of `SafeString | dict`

Reply:
Was wondering the same, and went the same way as `json_summary` in the end. I'll switch for both